stremio-core-web/src/model/serialize_ctx.rs

```diff
@@ -30,6 +30,15 @@ mod model {
         pub notifications: Notifications<'a>,
         pub search_history: Vec<SearchHistoryItem<'a>>,
         pub events: &'a Events,
+        pub streaming_server_urls: Vec<StreamingServerUrlItem>,
+    }
+
+    #[derive(Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct StreamingServerUrlItem {
+        pub url: String,
```

Comment:
- [ ] No need for a String here, Url is serialized to string:

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=80b575076c91b41ecd0443570bd9cedd